src/bundle/Resources/public/js/scripts/admin.contenttype.edit.js

```diff
@@ -70,6 +70,20 @@
             field.classList.toggle('ibexa-available-field-type--hidden', isFieldHidden);
         });
     };
+    const reinitIdentifierAutogenerator = (fieldNode) => {
```

Comment:
This reinit logic might be needed in other places. I think it would be better to add this to the original handler https://github.com/ibexa/admin-ui/blob/main/src/bundle/Resources/public/js/scripts/autogenerate.identifier.js by using some event to trigger it.